README.md

```diff
@@ -36,6 +36,10 @@ cd ../dynamo-api
 sam build
 gds aws <your-aws-account> -- sam deploy --parameter-overrides "AuthRegistrationBaseUrl=https://oidc.integration.account.gov.uk DynamoDbTableStackName=<the-name-you-chose>" --guided
 # Choose a decent stack name (name it after yourself for example) and accept the other defaults but tell it yes about lambdas that don't have authorisation
+
+To install and run DynamoDB local for session storage.
+Download and install Docker desktop - https://www.docker.com/products/docker-desktop/.
+cd ./backend/dynamo-db/ docker-compose up - This will spin up the local dynamoDB and switch SESSION_LOCAL flag true in .env file to use local DDB storage
```

Comment:
This is ambiguous I think - it reads like running `docker-compose up` will also set the flag to true but that's a separate step.  In either case I've suggested a change to the env var further down so check that before you change the README!